src/data/repositories/consts/AnalyticsConstants.ts

```diff
@@ -0,0 +1,221 @@
+export enum IndicatorsId {
+    suspectedDisease = "jLvbkuvPdZ6",
```

Comment:
I am guessing that these will be fetched from datastore in upcoming PRs?

Reply:
Yes!